### PR TITLE
Add toString for Event classes in SPI

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: presto-spi/src/main/java/io/prestosql/spi/eventlistener/SplitCompletedEvent.java [hunk 1]
```diff
@@ -99,4 +99,10 @@ public String getPayload()
     {
         return payload;
     }
+
+    @Override
+    public String toString()
+    {
+        return "SplitCompletedEvent[" + queryId + "," + stageId + "," + taskId + "]";
```
Comment (findepi) [inline presto-spi/src/main/java/io/prestosql/spi/eventlistener/SplitCompletedEvent.java:106]: no space after `,`?

also my IDE offers to generate sth like that
```
@Override
public String toString()
{
    return new StringJoiner(", ", SplitCompletedEvent.class.getSimpleName() + "[", "]")
            .add("queryId='" + queryId + "'")
            .add("stageId='" + stageId + "'")
            .add("taskId='" + taskId + "'")
            .toString();
}
```

File: presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryCreatedEvent.java [hunk 2]
```diff
@@ -45,4 +45,10 @@ public QueryMetadata getMetadata()
     {
         return metadata;
     }
+
+    @Override
+    public String toString()
+    {
+        return "QueryCreatedEvent[" + getMetadata().getQuery() + "]";
```
Comment (findepi) [inline presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryCreatedEvent.java:52]: queryId ?

File: presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryCompletedEvent.java [hunk 3]
```diff
@@ -100,4 +100,10 @@ public Instant getEndTime()
     {
         return endTime;
     }
+
+    @Override
+    public String toString()
+    {
+        return "QueryCompletedEvent[" + getMetadata().getQuery() + "," + getMetadata().getQueryState() + "]";
```
Comment (findepi) [inline presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryCompletedEvent.java:107]: queryId ?

**Review by findepi — APPROVED** [review]

File: presto-spi/src/main/java/io/prestosql/spi/eventlistener/SplitCompletedEvent.java (hunk 1 above)
Comment (losipiuk) [reply presto-spi/src/main/java/io/prestosql/spi/eventlistener/SplitCompletedEvent.java:106]: I can make it this way. A bit more verbse, but probably no harm

File: presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryCompletedEvent.java (hunk 3 above)
Comment (losipiuk) [reply presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryCompletedEvent.java]: Yeah - alrady noticed

**Comment by losipiuk** [comment]
AC

**Comment by losipiuk** [comment]
#5928

**Pull request merged**